vendor/knative.dev/pkg/controller/stats_reporter.go

```diff
@@ -198,7 +198,7 @@ func (r *reporter) ReportReconcile(duration time.Duration, success string, key t
 		return err
 	}
 
-	// TODO skonto: fix latency histogram
-	metrics.RecordBatch(ctx, reconcileCountStat.M(1))
+	metrics.RecordBatch(ctx, reconcileCountStat.M(1),
```

Comment:
Probably you want to revert that right? Same for all changes under knative.dev?

Reply:
uh. Thanks for catching this @skonto